src/microbe_stage/Patch.cs

```diff
@@ -516,6 +526,21 @@ public void ApplyVisibility(MapElementVisibility visibility)
             Region.Visibility = visibility;
     }
 
+    public void ApplyPatchNodeVisuals(WorldEffectVisuals visual)
+    {
+        activeWorldEffectVisuals.Add(visual);
+    }
+
+    public void ClearPatchNodeVisuals()
+    {
+        activeWorldEffectVisuals = new List<WorldEffectVisuals>();
```

Comment:
Why is a new list made here instead of the previous one cleared? This looks like a big mistake in terms of code efficiency. I'll fix this, but I'm commenting here for a learning opportunity.